src/h/fdefs.h

```diff
@@ -31,6 +31,9 @@ FncDefV(constructor)
 FncDef(copy,1)
 FncDef(cos,1)
 FncDef(cset,1)
+#ifdef DEVMODE
+FncDef(dbgbrk,0)
+#endif /* DEVMODE */
```

Comment:
Not a big deal, but the common format for the closing comment is to leave 5 tabs after endif.